packages/discord.js/src/structures/Message.js

```diff
@@ -674,6 +674,11 @@ class Message extends Base {
   /**
    * Fetches the Message this crosspost/reply/pin-add references, if available to the client
    * @returns {Promise<Message>}
+   * @example
+   * // Fetch the reference of this message
+   * message.fetchReference()
+   *   .then(reference => console.log(`Fetched the reference of this message: ${reference.content}`))
+   *   .catch(console.error);
```

Comment:
I don't think this example achieves anything. Methods that don't have options don't really demonstrate any usage. You're just calling the method.